.gitignore

```diff
@@ -12,4 +12,8 @@
 !unit_testing
 !.github/workflows
 !.gitignore
-!Makefile
\ No newline at end of file
+!Makefile
+!Dockerfile
+!docker
+!docker/doxygen
+!docker/multiple_tests
```

Comment:
Hm... mi a logika amogott, hogy minden blacklist, es csak azt whitelisteled, amit akarsz?  Csak kerdes. Marmint ertem, hogy mukodeik, de miert kenyelmesebb, atlathatobb igy, hogy majdnem mindent fel kell sorolni?

Reply:
Ez a kérdés minden egyes gitignore bővítés után felmerült bennem is és pont amennyire tűnik butaságnak, annyi racionalitást tudok elfogadni mögötte. 
Az ötlet eredetileg szintén nem saját, viszont nekem is azt tanították szakközépsuliban, főleg hálózatok (portok, tűzfal témakörén belül) órán, hogy sokkal biztonságosabb mindent tiltani és csak azt engedélyezni, amit szabad, mint mindent engedélyezni és azt tiltani, amit nem.

Reply:
Ha az üzemeltető kifelejt valamit, aminek át kéne mennie egy folyamaton, inkább találkozik egy hibaüzenettel és elszáll a program, ha viszont valamit lefedni felejt ki abból lehetnek csúnya gondok is, breach, injection, egyéb anomáliák kíváncsi "felhasználók" jóvoltából 😄 Jelen esetben a vs és vscode volt amitől féltem, hogy teledobálja furcsa temp fájlokkal és utólag kell korrigálni mindig

Reply:
@hegyhati a tag kimaradt

Reply:
Tuzfalnal igen, ez egertelmu :-) C++ kodban is azt mondanam, hogy legyen minden const, es kulon ami nem az, arrol vegyuk le, tiltva legyen mindenhol a copy, stb. Itt nem latom ezt annyira indokoltnak. Ha  valami veletlenul belekerul, egy git rm-mel ki lehet szedni a kovetkezo commitban. Ha ceges titok, akkor git reset HEAD. Szerintem az nagyobb problema, hogy valamit veletlenul elfelejtesz hozzaadni, felpusholsz, es mondjuk nincs ott a header melle a cpp. De ha neked igy jo, nem szolok bele :-)